e2e/_suites/helm/helm_charts_test.go

```diff
@@ -81,13 +83,12 @@ func setupSuite() {
 	timeoutFactor = shell.GetEnvInteger("TIMEOUT_FACTOR", timeoutFactor)
 
 	stackVersion = shell.GetEnv("STACK_VERSION", stackVersion)
-	stackVersion = e2e.GetElasticArtifactVersion(stackVersion)
+	stackVersion = utils.GetElasticArtifactVersion(stackVersion)
 
-	h, err := k8s.HelmFactory(helmVersion)
+	_, err := helm.Factory(helmVersion)
 	if err != nil {
 		log.Fatalf("Helm could not be initialised: %v", err)
 	}
-	helm = h
```

Comment:
```suggestion
h, err := helm.Factory(helmVersion)
if err != nil {
	log.Fatalf("Helm could not be initialised: %v", err)
}
helmManager = h
```
I think this would resolve the panic caused by not initialising the helm manager.